.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,21 +4,21 @@
 >
 > We've designed this PR template to speed up the PR review and merge process - please use it.
 
-> _Please describe the change here._ 
+## WHAT is this change about?
+> _Please describe the change here_
 
----
+## Does this PR introduce a change to `config/values.yml`?
+> _Simply answer Yes or No. You can also review [this "cf-for-k8s values for contributors" doc](https://docs.google.com/document/d/1Y3jAx48TCGIQdzOFmzqp_R_0WT8Hfjx9oyqs2Tk0Otw/edit#) for up-to-date principles and guidelines for contributing values changes_
 
-- Make sure this PR is based off the `develop` branch
-- If you're adding/removing/changing any values in `config/values.yml`, please review [this "cf-for-k8s values for contributors" doc](https://docs.google.com/document/d/1Y3jAx48TCGIQdzOFmzqp_R_0WT8Hfjx9oyqs2Tk0Otw/edit#) for up-to-date principles and guidelines for contributing values changes.
-- Checkout the [contributing guidelines](https://github.com/cloudfoundry/cf-for-k8s/blob/develop/community/CONTRIBUTING.md)
-- Include any links to related PRs, issues, stories, slack discussions, etc... that will help establish context.
-- Is there anything else of note that the reviewers should know about this change?
-
-**Acceptance Steps**
+## Acceptance Steps
+> _Please replace this with a series of instructions (eg kubectl or cf cli commands) for how our Product Manager can verify that your changes were properly integrated._
 
-_please replace this with a series of instructions (eg kubectl or cf cli commands) for how our Product Manager can verify that your changes were properly integrated_
+## Tag your pair, your PM, and/or team
+> _Optional but it's helpful to tag a few other folks on your team or your team alias in case we need to follow up later._
 
 
-_Tag your pair, your PM, and/or team_
-
-> _It's helpful to tag a few other folks on your team or your team alias in case we need to follow up later._
+## Things to remember
+- Make sure this PR is based off the `develop` branch
+- Include any links to related PRs, issues, stories, slack discussions, etc... that will help establish context.
+- Is there anything else of note that the reviewers should know about this change?
+- Checkout the [contributing guidelines](https://github.com/cloudfoundry/cf-for-k8s/blob/develop/community/CONTRIBUTING.md)
```

Comment:
let's add a bullet right above this one that says "Our CI uses concourse task files from HEAD of the `develop` branch. If your PR includes/requires CI changes, please ping relint interrupt and they'll help apply the CI changes for you."